src/main/java/com/connexta/search/common/configs/SolrClientConfiguration.java

```diff
@@ -1,17 +0,0 @@
-/*
```

Comment:
I thought you deleted that originally. I don't remember removing this.

Reply:
🤦‍♀ I can add it back in.

Reply:
Actually, I remember why I made it like this originally. I was thinking that we might have a different kind of solr URL, like one that uses https instead of http. I'll actually simplify the configuration to take just a URL string.